parm/atm/variational/3dvar_dripcg.yaml

```diff
@@ -2,6 +2,7 @@ cost function:
   cost type: 3D-Var
   window begin: '{{ ATM_WINDOW_BEGIN | to_isotime }}'
   window length: $(ATM_WINDOW_LENGTH)
+  window shift: true
```

Comment:
This syntax has changed in develop this week. Just noting that here.

Reply:
Noted!!  Thank you.  

current configuration:
```
window begin: "2018-04-14T23:30:00Z"
window end: "2018-04-15T03:00:00Z"
window shift: true
```

new configuration:
```
time window:
  begin: "2018-04-14T23:30:00Z"
  end: "2018-04-15T03:00:00Z"
  bound to include: begin
```

Looks like we will have to update the window section in the YAMLs when we merge develop branches from JCSDA.

Reply:
It's already in GDASApp develop. We need to discuss about when to merge this sprint branch into develop of GDASApp. I would suggest doing so as soon as possible. We should chat with @ADCollard and @RussTreadon-NOAA

Reply:
> It's already in GDASApp develop. We need to discuss about when to merge this sprint branch into develop of GDASApp. I would suggest doing so as soon as possible. We should chat with @ADCollard and @RussTreadon-NOAA

@CoryMartin-NOAA Is there any reason not to do this?

Reply:
Russ and I want to, we just don't want to if there is a reason not to. The sprint can continue, we were just wanting to get what is working into develop "soon"